pkg/codegen/go/gen.go

```diff
@@ -1067,6 +1060,19 @@ func GeneratePackage(tool string, pkg *schema.Package) (map[string][]byte, error
 		}
 	}
 
+	return packages
+}
+
+func GeneratePackage(tool string, pkg *schema.Package) (map[string][]byte, error) {
+	var goInfo GoInfo
```

Comment:
Nothing really changed in this function other than factoring out some stuff that the resource docs generator also needs access to via the `DocLangHelper` interface.